week-5/Homework/mandatory/1-exercises/exercises.js

```diff
@@ -0,0 +1,164 @@
+/**
+ *
+ * For each of the names in the array passed into this function
+ *
+ * - Add a <h1> tag to the website containing the name of the person
+ * - Add a <h2> tag to the website containing the job of the person
+ *
+ * All of your HTML should go inside the Div tag with the id "content".
+ *
+ * <div id="content">
+ *      <h1>{Name Here}</h1>
+ *      <h2>{Job Here}</h2>
+ *      .....
+ * </div>
+ */
+function exerciseOne(arrayOfPeople) {
+  let content = document.querySelector("#content");
+  arrayOfPeople.forEach((element) => {
+    hOnes = document.createElement("h1");
+    hOnes.innerHTML = element.name;
+    hTwoes = document.createElement("h2");
+    hTwoes.innerHTML = element.job;
+    let myDiv = document.querySelector("#content");
+    myDiv.appendChild(hOnes);
+    myDiv.appendChild(hTwoes);
+  });
+}
+
```

Comment:
Make sure that you use an appropriate keyword when declaring a variable for the first time within a given scope - i.e. `var`, `let` or `const`. If you don't have any such keyword then JavaScript will automatically assume that you want the variable initialised with global scope or that you want to update an existing variable accessible within scope. This can be a problem since you can accidentally overwrite data that's needed somewhere else in your code.

So, for example, in `exerciseTwo()`:

```js
const ulEl = document.createElement(ul);
```

Note that it is OK to use `const` here since `const` does not prevent you from modifying the object, such as to, it simply prevents you assigning another value to that variable identifier (`ulEl`) within the same scope. This might be a good thing since it means if you were to unintentionally do this later in your code then you would get an error.